opentech/apply/users/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from wagtail.wagtailusers.forms import UserEditForm, UserCreationForm
+
+
+class CustomUserEditForm(UserEditForm):
```

Comment:
Could you not just pull this out into a common class and then both forms inherit from it. 

would also prefer to be explicit about the content of the input: "Not used - see full_name," rather than something a little cryptic

Reply:
changed the contents of the input.

need to have the init in each class. Doing it on a common class doesn't make the change

Reply:
sorted, thanks!